types/devicefarmer__stf-syrup/index.d.ts

```diff
@@ -0,0 +1,57 @@
+import Bluebird = require("bluebird");
+
+type extractDesRet<RetT> = RetT extends SyrupI<never, never, infer RetX> ? RetX
+    : unknown;
+type extractBluebirdReturnR<RetT> = RetT extends Bluebird<infer RetX> ? RetX
+    : RetT;
+declare class SyrupI<
+    OptionsT extends object = any, // TODO: find a way to remove any. Maybe we union all the options that are needed for each dependency?
+    DepsT extends SyrupI[] = [],
+    // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
```

Comment:
Why disable these rules? Is unknown not sufficient?

Reply:
Works without them. I guess I'll delete them